cookiecutter.json

```diff
@@ -10,5 +10,25 @@
     "use_flower": "{% if cookiecutter.use_celery == 'y' %}y{% else %}n{% endif %}",
     "celery_flower_user": "{{(['QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm1234567890']*16)|map('random')|join('')}}",
     "celery_flower_password": "{{(['QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm1234567890']*16)|map('random')|join('')}}",
-    "sentry_dsn": "https://<key>@sentry.io/<project>"
+    "sentry_dsn": "https://<key>@sentry.io/<project>",
+    "https_redirect": "n",
+    "https_proxy_header": "X-Scheme",
+    "csp_enabled": "y",
+    "csp_report_only": "y",
+    "csp_report_url": "https://sentry.io/api/-1/security/?sentry_key=<key>",
+    "csp_default_src": "'none'",
```

Comment:
No need of single quote inside double quote. environment variables don't need quotes.

Reply:
It is needed as `'none'` is a special value and it has to be in single quotes (see [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy#Multiple_content_security_policies)).

Reply:
Why `'none'` is a special value? You can just keep `none` as that special value and then use quotes when using the value.